terraform/modules/emblem-app/main.tf

```diff
@@ -78,3 +78,102 @@ resource "google_storage_bucket_iam_member" "sessions-iam" {
   role   = "roles/storage.objectAdmin"
   member = "serviceAccount:${google_service_account.website_manager.email}"
 }
+
+############################
+# E2E Delivery testing IAM #
+############################
```

Comment:
I think there's some need for clarity on which module this should live in, for now let's put it in a dedicated delivery_testing.tf file and we'll sort the rest in follow-up.

Reply:
👍 done.